packages/graphql-api/src/modules/accounts-password/index.ts

```diff
@@ -0,0 +1,43 @@
+import { GraphQLModule } from '@graphql-modules/core';
+import { AccountsPassword } from '@accounts/password';
+import TypesTypeDefs from './schema/types';
+import getQueryTypeDefs from './schema/query';
+import getMutationTypeDefs from './schema/mutation';
+import getSchemaDef from './schema/schema-def';
+import { Query } from './resolvers/query';
+import { Mutation } from './resolvers/mutation';
+import { IAccountsRequest } from '../accounts';
+import { mergeGraphQLSchemas } from '@graphql-modules/epoxy';
+
+export interface IAccountsPasswordModuleConfig {
```

Comment:
Can you remove the `I` in front of the types please? (to keep consistency with the code we already have))

Reply:
Done :)